go.mod

```diff
@@ -3,23 +3,20 @@ module github.com/stelligent/config-lint
 go 1.12
 
 require (
-	github.com/apparentlymart/go-cidr v1.0.1
+	github.com/Azure/go-autorest v12.2.0+incompatible // manually added to fix multiple modules error from goreleaser
```

Comment:
This was added to address an issue with running the `goreleaser release` command.

Fixes this error:
```
build github.com/goreleaser/goreleaser: cannot load github.com/Azure/go-autorest/autorest: ambiguous import: found github.com/Azure/go-autorest/autorest in multiple modules:
	github.com/Azure/go-autorest v12.0.0+incompatible (/go/pkg/mod/github.com/!azure/go-autorest@v12.0.0+incompatible/autorest)
	github.com/Azure/go-autorest/autorest v0.9.2 (/go/pkg/mod/github.com/!azure/go-autorest/autorest@v0.9.2)
##[error]Process completed with exit code 1.
```